### ISSUE TITLE
Version 0.8.1 is incompatible with Boost 1.70.0

### ISSUE THREAD

**Issue by gjasny** [zaphoyd/websocketpp#808]
Hello,

while trying out Boost 1.70.0 I noticed that websocketpp does not build anymore. I'm getting errors like:
```
[  1%] Building CXX object examples/associative_storage/CMakeFiles/associative_storage.dir/associative_storage.cpp.o
In file included from /Users/gjasny/src/websocketpp/examples/associative_storage/associative_storage.cpp:4:
In file included from /Users/gjasny/src/websocketpp/websocketpp/config/asio_no_tls.hpp:32:
In file included from /Users/gjasny/src/websocketpp/websocketpp/transport/asio/endpoint.hpp:32:
In file included from /Users/gjasny/src/websocketpp/websocketpp/transport/asio/connection.hpp:31:
In file included from /Users/gjasny/src/websocketpp/websocketpp/transport/asio/base.hpp:31:
In file included from /Users/gjasny/src/websocketpp/websocketpp/common/asio.hpp:61:
In file included from /Users/gjasny/Git/ExternalLibs/boost/boost/asio/steady_timer.hpp:22:
In file included from /Users/gjasny/Git/ExternalLibs/boost/boost/asio/basic_waitable_timer.hpp:27:
In file included from /Users/gjasny/Git/ExternalLibs/boost/boost/asio/executor.hpp:338:
/Users/gjasny/Git/ExternalLibs/boost/boost/asio/impl/executor.hpp:179:22: error: no member named 'context' in 'std::__1::reference_wrapper<boost::asio::io_context>'
    return executor_.context();
           ~~~~~~~~~ ^
``` 

@vinniefalco Do you have some hints how to port code to latest ASIO? Or is there an ASIO changelog for Boost 1.70?

**Comment by vinniefalco** [comment]
Yes, I believe this is the relevant commit:
https://github.com/boostorg/asio/commit/59066d80b26e1d5b83b60d127ee17948d9ae9702

**Comment by mharmer** [comment]
Duplicate of #794.

**Comment by zaphoyd** [comment]
The fix for this issue has been available on the `develop` branch for some time and has now been published in release 0.8.2
